MAUI/DataGrid/export-to-excel.md

```diff
@@ -0,0 +1,548 @@
+                </syncfusion:DataGridNumericColumn>
+                <syncfusion:DataGridTextColumn HeaderText="Customer ID"
+                                               MappingName="CustomerID">
+                </syncfusion:DataGridTextColumn>
+                <syncfusion:DataGridTextColumn MappingName="Customer"
+                                               HeaderText="Customer">
+                </syncfusion:DataGridTextColumn>
+                <syncfusion:DataGridTextColumn HeaderText="Ship City"
+                                               MappingName="ShipCity">
+                </syncfusion:DataGridTextColumn>
+                <syncfusion:DataGridTextColumn HeaderText="Ship Country"
+                                               MappingName="ShipCountry">
+                </syncfusion:DataGridTextColumn>
+            </syncfusion:SfDataGrid.Columns>
+        </syncfusion:SfDataGrid>
+    </StackLayout>
+{% endhighlight %}
+{% highlight c# %}
+private void ExportToExcel_Clicked(object sender, EventArgs e)
+{
+    // Perform exporting to Excel sheet operations here.
+    DataGridExcelExportingController excelExport = new DataGridExcelExportingController();
+    var excelEngine = excelExport.ExportToExcel(this.dataGrid);
+    var workbook = excelEngine.Excel.Workbooks[0];
+    MemoryStream stream = new MemoryStream();
+    workbook.SaveAs(stream);
+    workbook.Close();
+    excelEngine.Dispose();
+
+    string OutputFilename = "DefaultDataGrid.xlsx";
+    SaveService saveService = new();
+    saveService.SaveAndView(OutputFilename, "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet", stream);
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Export DataGrid to Excel format" src="Images\Export-to-Excel\maui-datagrid-datagrid-to-excel.png" width="689"/>
+
+N> SfDataGrid cannot export the DataGridTemplateColumn to PDF or Excel,  since we cannot get the loaded views and draw them with the particular range, values etc from DataGridTemplateColumn.
+
+## Exporting Options
+
+Additionally, you can export the data to Excel by providing the grid and `DataGridExcelExportingOption` as arguments to the `ExportToExcel` method, which offers several customization options.
+
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingController excelExport = new DataGridExcelExportingController();
+DataGridExcelExportingOption exportOption = new DataGridExcelExportingOption();
+exportOption.CanExportColumnWidth = false;
+exportOption.DefaultColumnWidth = 150;
+var excelEngine = excelExport.ExportToExcel(this.dataGrid, exportOption);
+{% endhighlight %}
+{% endtabs %}
+
+The SfDataGrid provides several properties in `DataGridExcelExportingOption` class to customize the grid while exporting it to Excel.
+
+### Exporting formatted text and actual value
+
+By default, the actual value will only be exported to Excel. To export the display text, set the ExportMode property as Text.
```

Comment:
The actual value will be exported to Excel by default. To export the display text, you need to set the ExportMode property as Text.